src/llvm_backend.hpp

```diff
@@ -1,12 +1,12 @@
-#if defined(GB_SYSTEM_WINDOWS)
-#include "llvm-c/Core.h"
-#include "llvm-c/ExecutionEngine.h"
-#include "llvm-c/Target.h"
-#include "llvm-c/Analysis.h"
-#include "llvm-c/Object.h"
-#include "llvm-c/BitWriter.h"
-#include "llvm-c/DebugInfo.h"
-#include "llvm-c/Transforms/PassBuilder.h"
+#if defined(GB_SYSTEM_WINDOWS) || defined(GB_SYSTEM_OSX)
+#include <llvm-c/Core.h>
+#include <llvm-c/ExecutionEngine.h>
+#include <llvm-c/Target.h>
+#include <llvm-c/Analysis.h>
+#include <llvm-c/Object.h>
+#include <llvm-c/BitWriter.h>
+#include <llvm-c/DebugInfo.h>
+#include <llvm-c/Transforms/PassBuilder.h>
```

Comment:
They were local imports on Windows for a reason. This was not a mistake.

Reply:
Right. That makes sense. This is very much still in-progress. I'll fix this.